### ISSUE TITLE
[FEATURE REQUEST] Left Hand

### ISSUE THREAD

**Issue by parrot-m** [flarialmc/dll#80]
### Feature description

I would really like to see a feature to enable left hand with items following it. Thank you.

### Reason

This feature should be added because it would spice up a lot of peoples gameplay.

**Comment by treegfx** [comment]
so basically viewmodel 🤯

**Comment by PoopyHaters** [comment]
hi tree

**Comment by PoopyHaters** [comment]
check my discord pfp

**Comment by parrot-m** [comment]
> so basically viewmodel 🤯

Yeah but you cant change the orientation of the blocks to be towards the center and viewmodel just zooms it in or out
